internal/server/api_restore.go

```diff
@@ -51,11 +51,13 @@ func handleRestore(ctx context.Context, rc requestContext) (interface{}, *apiErr
 
 	switch {
 	case req.Filesystem != nil:
-		out := req.Filesystem
```

Comment:
hmm, the shadow linter did not detect this. ?

Reply:
yeah, this is weird. those linters are finicky.